README.md

```diff
@@ -166,5 +167,9 @@ The -throughput -1 specifies the writes tries to write the events at the maximum
 ```
 
 ### Recording the latencies to CSV files
-user can use the options "-writecsv  <file name>" to record the latencies of writers and "-readcsv <file name>" for readers.
+User can use the options "-writecsv  <file name>" to record the latencies of writers and "-readcsv <file name>" for readers.
 in case of End to End latency mode, if the user can supply only -readcsv to get the end to end latency in to the csv file.
+
+### Kafka Benchmarking
+User can set the option "-kafka true"  for Kafka Benchmarking. User should create the topics manually before running this for kafka benchmarking.
+Unlike pravega benchmarking, this tool does not create the topic automatically. This tools treats stream name as a topic name.
```

Comment:
If "Kafka" and "Pravega" are written with caps before, we need to keep the style consistent everywhere else.